### PR TITLE
Enable paraview catalyst

### PR DESCRIPTION
**Pull request by tjotaha**
The changes are in Realm.C and OutputInfo.C, and OutputInfo.h. The build will succeed as long as you are building against the Trilinos master branch and you set the following flag in your Trilinos build:

-DTrilinos_ENABLE_SEACASIoss:BOOL=ON

ParaView Catalyst will only be active when an "output" block in a Nalu input deck contains a "catalyst_file_name" command. It is also necessary to build the ParaView Catalyst adapter against an install of ParaView and set an environment variable to point to the install location. We will need to update the Nalu build instructions to show these details. 

The environment variable is called CATALYST_ADAPTER_INSTALL_DIR. It should be set to the root of the install directory for the Catalyst adapter.

### CODE REVIEW

File: src/Realm.C [hunk 1]
```diff
@@ -1928,7 +1942,46 @@ Realm::create_output_mesh()
       if (fileid++ > 0) oname += "-s" + fileid_ss.str();
     }
 
-    resultsFileIndex_ = ioBroker_->create_output_mesh( oname, stk::io::WRITE_RESULTS, *outputInfo_->outputPropertyManager_);
+
+    if(!outputInfo_->catalystFileName_.empty()||
```
Comment (spdomin) [inline src/Realm.C]: Is the concept here that one obtains either Catalyst **or** standard output? Is there a manner to support both in a single simulation? I think that would be most preferable.

Comment (tjotaha) [inline src/Realm.C]: Yes, the implementation either writes to Catalyst or to standard output but not to both. In Sierra, we duplicate a results output block in the input deck, and have one block for Catalyst and one block for Exodus II output, if desired. In Nalu you can only have one output section in the input deck?

Comment (spdomin) [inline src/Realm.C]: In theory, we should be able to have multiple blocks without too much issue. For now, we can move with the either/or path and clean up the I/O to allow for a general amount of output blocks. At present, we support one unique output and restart block.

**Comment by spdomin** [comment]
@tjotaha, can you please add a pull request to modify the Trilinos configure file?

**Pull request merged**